libr/include/r_cons.h

```diff
@@ -1037,6 +1037,7 @@ typedef struct r_panels_t {
 	bool isZoom;
 	RPanelsMenu *panelsMenu;
 	Sdb *db;
+	Sdb *mdb;
```

Comment:
Why don't you just use a `SdbHash` (which is an hashtable)? This is what, it seems, you really need here. You don't need all the things of `sdb`, but you just want to store a number for a given string key.

Reply:
woo thanks and that's exactly it.

...tbh i didn't even know such a thing, my bad. i will fix this using that shortly later :)

Reply:
It's still not perfect in its implementation and API (it only supports `char *` as key for now) but it will be better soon :)

Reply:
cool and that should be enough for me anyway here :)